cacerts/tests/test.bats

```diff
@@ -0,0 +1,4 @@
+@test "cert.pem contains a valid Mozilla certificate" {
+  _output="$(cat "/hab/pkgs/${TEST_PKG_IDENT}/ssl/cert.pem")"
+   [[ "${_output}" =~ 'Certificate data from Mozilla' ]] || failWithMessage "${_output}"
```

Comment:
Same comment as on other PR:

https://github.com/habitat-sh/core-plans/pull/2665/files#r294322796

...`[[` should be avoided.

Reply:
Hi @james-stocks.  In the case when using the "=~" match operation, the double brackets are required.  There are a number of other core-plans that use this match operation including 7zip, apr, etc.  In this case would you be ok to keep the double brackets?

Reply:
Hi @james-stocks.  Waiting on the outcome from https://github.com/habitat-sh/core-plans/issues/2716 before taking action here, to remove or keep the '[['.

Reply:
Replaced the '[[' with '[ ]'  + grep to produce the following:

* on success

```bash
 ✓ cert.pem contains a valid Mozilla certificate

1 test, 0 failures
```


* on failure a useful error message is returned something like:

```bash
 ✗ cert.pem contains a valid Mozilla certificate
   (in test file tests/test.bats, line 5)
     `diff <(echo "${actual}") <(echo "${expected}")' failed
   1c1
   < ERROR: did not find the expected text below
   ---
   > Certificate data from Mozilla

1 test, 1 failure
```